sdk/src/main/java/com/mapbox/maps/Style.kt

```diff
@@ -324,6 +324,15 @@ class Style internal constructor(
       this.setStyleLightProperty(id, light)
     }
 
+  /**
+   * Removes light from style if it was set.
+   */
+  fun removeStyleLight() {
```

Comment:
```suggestion
  fun removeLight() {
```

Reply:
we have `setStyleLight` method so I've selected `removeStyleLight` to be consistent with setter, @mapbox/maps-android WDYT?

Reply:
setStyle* is the naming for the low level style API, in high level APIs we remove the `Style` from the function name.

And just double checked, we should add the removeLight and removeTerrain API as extension functions to the Light and Terrian class, as the higher level add APIs are extended that way as well.

Reply:
@pengdev thanks, fixed!